addons/medical_feedback/functions/fnc_effectBleeding.sqf

```diff
@@ -5,7 +5,7 @@
  *
  * Arguments:
  * 0: Enable effect <BOOL>
- * 1: Current bloodloss (in l/s) <NUMBER>
+ * 1: Current bloodloss (in l/s) <NUMBER> (Optional if enable is false)
```

Comment:
```suggestion
 * 1: Current bloodloss (in l/s) <NUMBER> (optional if enable is false)
```

Reply:
I think `optional` can remain as a keyword in header documentation, in the way it's used here, but it should be documented.